CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ set(CHPLX_VERSION
 )
 
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_FLAGS_INIT "-std=c++20")
```

Comment:
Why do we need this?

Reply:
without that flag cmake slips in a compile time flag (similar to experimental) that breaks the build.